ts/details.ts

```diff
@@ -74,6 +74,11 @@ switch (tableContainer.getAttribute("data-detail-type")) {
     }
     const copyButton = makeCopyButton(requisitionName);
     titleMisoLink.parentNode?.insertBefore(copyButton, titleMisoLink);
+    const qcReportButton = document.getElementById("qcReportButton");
+    if (qcReportButton) {
+      qcReportButton.style.display = "none";
+    }
+    break;
```

Comment:
I think the `break` statement alone is the fix here. The button is created below, and without an ID, so I don't think the rest of this is doing anything?

Please add another `break` statement at the end of the `CASE_ID` case to prevent similar bugs if we expand this in the future.